### PR TITLE
Neutralizer/elk stack addition

### PR DESCRIPTION
**Pull request by Neutralizer**
ELK addition in separate container with logback

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -31,4 +31,15 @@ services:
...
       - "3306:3306"
     expose:
-      - "3306"
\ No newline at end of file
+      - "3306"
+
+  #5601 (Kibana web interface).
+  #9200 (Elasticsearch interface).
+  elk:
+    image: neutralizer-test/elk
```
Comment (matrixuser87) [inline docker-compose.yml]: use more descriptive name

File: pom.xml [hunk 2]
```diff
@@ -89,6 +89,23 @@
             <version>8.0.18</version>
         </dependency>
 
+<!--        <dependency>-->
```
Comment (matrixuser87) [inline pom.xml]: remove the commented lines

File: src/main/java/com/controller/QuoteController.java [hunk 3]
```diff
@@ -128,4 +134,28 @@ public ResponseEntity dislikeQuote(@PathVariable int id) {
...
+        LOG.error(response);
+        return new ResponseEntity<>(response, HttpStatus.OK);
+    }
+
+
+    @GetMapping(path = "/exception")
+    public ResponseEntity error() {
+        String errorResponse = "";
+        try {
+            throw new Exception("Test exception");
```
Comment (matrixuser87) [inline src/main/java/com/controller/QuoteController.java:151]: if this is a test, move it to the tests

**Review by matrixuser87 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (Neutralizer) [reply pom.xml]: removed

File: docker-compose.yml (hunk 1 above)
Comment (Neutralizer) [reply docker-compose.yml]: name changed

File: src/main/java/com/controller/QuoteController.java (hunk 3 above)
Comment (Neutralizer) [reply src/main/java/com/controller/QuoteController.java:151]: Integration tests will be implemented at a later date.

**Pull request merged**